### DISCUSSION TITLE
Pyparsing 3.2.0b1

### DISCUSSION THREAD

**Discussion by ptmcg** [pyparsing/pyparsing#569] [Announcements]
Major feature is dropping support for Python 3.6-3.8.
Also includes some changes to exceptions raised when parsing fails.

Please report any success/failure in using your project with this version.

**Comment by Argmaster** [comment]
Why 3.8 too? Is there known incompatibility?

**Reply by Argmaster** [reply]
Ok, I see release notes, adoption of new features. I get that.

**Reply by ptmcg** [reply]
Plus 3.8 is coming right up on its end-of-life - if there are folks out there still running on 3.8, they can just pin pyparsing to <3.2. That way, if I _do_ backport anything to a 3.1.x branch (unlikely but not impossible), they would still get it.

THanks for checking out the beta!

**Reply by Argmaster** [reply]
Yeah, I will probably pin <3.2 for my library, thanks!
